setup.gradle

```diff
@@ -54,6 +54,11 @@ ext.setupArrowModule = {
 
 ext {
     excludeArrow = { exclude group: "io.arrow-kt" }
+    excludeOldKotlinVersions = {
+        exclude group: "org.jetbrains.kotlin", module: "kotlin-reflect"
+        exclude group: "org.jetbrains.kotlin", module: "kotlin-stdlib"
```

Comment:
I don't know if it would be interesting to add the exclusions and then do the refactor with this configuration, because maybe it's necessary to exclude other modules...

Reply:
So far thats what I found until now.. and I will have to use that within core at least. If we find more, we will be updating little by little.

Reply:
about
```
        exclude group: "org.jetbrains.kotlin", module: "kotlin-stdlib-jdk7"
        exclude group: "org.jetbrains.kotlin", module: "kotlin-stdlib-jdk8"
```
I think both are needed to keep Android happy.. @aballano shared this the other day:
https://medium.com/@mbonnin/the-different-kotlin-stdlibs-explained-83d7c6bf293

Reply:
Using the jdk7 or, in other words, not using jdk8 is more a recommendation, we can use it, but if any of the APIs not available for Android are used, then clients might have a crash. Although we have animalSniffer for that as well for that, so I don't think it matters that much

Reply:
Right, they are necessary. When excluding them, it's possible to control which versions are being used. There are `org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.61` and `org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.70` for instance. If `kotlin-stdlib` is being excluded, maybe those should be excluded as well.

Reply:
The idea is excluding and adding the required versions.

Reply:
ahh I see now!! 👍